pages/en/release-notes/graphql-validations-migration-guide.mdx

```diff
@@ -0,0 +1,536 @@
+---
+title: GraphQL Validations migration guide
+---
+
+Soon `graph-node` will support 100% coverage of the [GraphQL Validations specification](https://spec.graphql.org/June2018/#sec-Validation).
+
+Previous versions of `graph-node` did not support all validations and provided more graceful responses - so, in cases of ambiguity, `graph-node` was ignoring invalid GraphQL operations components.
+
+GraphQL Validations support is the pillar for the upcoming new features and the performance at scale of The Graph Network.
+
+It will also ensure determinism of query responses, a key requirement on The Graph Network.
+
+**Enabling the GraphQL Validations will break some existing queries** sent to The Graph API.
+
+To be compliant with those validations, please follow the migration guide.
+
+> ⚠️ If you do not migrate your queries before the validations are rolled out, they will return errors and possibly break your frontends/clients.
+
+# Migration guide
+
+## Migration CLI tool
+
+**Most of the GraphQL operations errors can be found in your codebase ahead of time.**
+
+For this reason, we provide a smooth experience for validating your GraphQL operations during development or in CI.
+
+`[@graphql-validate/cli](https://github.com/saihaj/graphql-validate)` is a simple CLI tool that helps validate GraphQL operations against a given schema.
+
+### **Getting started**
+
+You can run the tool as follows:
+
+```bash
+npx @graphql-validate/cli -s https://api-next.thegraph.com/subgraphs/name/$GITHUB_USER/$SUBGRAPH_SLUG -o *.graphql
+```
+
+**Notes:**
+
+- Set or replace $GITHUB_USER, $SUBGRAPH_SLUG with the appropriate values. Like: [`artblocks/art-blocks`](https://api.thegraph.com/subgraphs/name/artblocks/art-blocks)
+- The provided preview schema URL (`https://api-next.thegraph.com/`) is heavily rated limited and will sunset on [deadline date]. **Do not use it in production.**
+- Operations are identified in files with the following extensions `[.graphql,](https://www.graphql-tools.com/docs/schema-loading#graphql-file-loader)[.ts, .tsx, .js, jsx](https://www.graphql-tools.com/docs/schema-loading#code-file-loader)` (`-o` option).
+
+### CLI output
+
+The `[@graphql-validate/cli](https://github.com/saihaj/graphql-validate)` CLI tool will output the GraphQL operations errors as follows:
+
+![Untitled](GraphQL%20Validations%20migration%20guide%20ba6225eb33e54d0e88907d7abaa50a5e/Untitled.png)
+
+For each error, you will find a description, file path and position, and a link to a solution example (see the following section).
+
+## Run your local queries against the preview schema
+
+We provide an endpoint `https://api-next.thegraph.com/` that runs a `graph-node` version that has validations turned on.
```

Comment:
I think https://api-next.thegraph.com/ needs to be reactivated ahead of merging this cc @lutter @schmidsi